examples/wcf/shared/Examples.Wcf.Shared.csproj

```diff
@@ -1,5 +1,4 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
     <TargetFrameworks>net8.0;net462</TargetFrameworks>
```

Comment:
Should the `net8.0` be replaced with a property?

Reply:
Done https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2087/commits/a6833f7a8b533e6259c8780d550aa6caa2e2baa8.